usaspending_api/search/migrations/0041_alter_mv_agency_office_autocomplete_unique_constraint.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.23 on 2024-02-08 05:35
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('search', '0040_initial_mv_agency_office_autocomplete'),
+    ]
+    
+    operations = [
+        migrations.RunSQL(
+            """
+            ALTER TABLE mv_agency_office_autocomplete
+            ADD CONSTRAINT mv_agency_office_autocomplete_unique_constraint
+            UNIQUE (toptier_code, toptier_abbreviation, toptier_name,
+                    subtier_abbreviation, subtier_name, subtier_code,
+                    office_code, office_name);
+            """
+        ),
```

Comment:
Might want to add a `reverse_sql` parameter to the `RunSQL()` function that will remove the constraint in case anyone needs to rollback the PR.